packages/gestalt/src/Masonry.js

```diff
@@ -41,6 +47,9 @@ type Props<T> = {|
   items: Array<T>,
   measurementStore: Cache<T, *>,
   minCols: number,
+  // Content layer and Viewport layer is as defined in Collection.
+  onLayerUpdate?: (contentLayer: Layer, viewportLayer: Layer) => void,
```

Comment:
Again, I think the concept of a "layer" here is internal to Masonry and not very obvious from the outside. How about `onVirtualizationUpdate`, `onBoundsUpdate`, `onWindowUpdate`?

Reply:
"Layer" type info is duplicated in arg names too. `content: Layer, viewport: Layer`.

Reply:
done. updated name to onVirtualizationWindowUpdate